### PR TITLE
VAR-39 | Fixed time zone

### PR DESCRIPTION
**Pull request by Nizar-Rahme**

### CODE REVIEW

File: app/utils/timeUtils.js [hunk 1]
```diff
@@ -225,6 +225,26 @@ function getTimeDiff(startTime, endTime, unit, isFloat = false) {
...
 }
 
+/**
+ * getDateWithinTZ
+ * Get Date object with the default time-zone factored in
+ *
+ * @param {Moment | Date | string | undefined} value represents a specific date or undefined for the current moment
+ * @returns {Date} date object correspondent to the supplied value
+ */
+function getDateWithinTZ(value) {
```
Comment (rBoost) [inline app/utils/timeUtils.js]: Why do we need this function? Why not `moment().toDate()`? https://momentjs.com/docs/#/displaying/as-javascript-date/

Comment (midori0507) [reply app/utils/timeUtils.js]: Same question

Comment (Nizar-Rahme) [reply app/utils/timeUtils.js]: https://github.com/moment/moment-timezone/issues/309#issuecomment-193885034

**Comment by midori0507** [comment]
Close for unnecessary and will re-open when needed

**Pull request closed without merging**